v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/helpers/EntityHelperImpl.java

```diff
@@ -170,8 +162,9 @@ public void setRiptide(Entity entity, boolean state) {
     @Override
     public void forceInteraction(Player player, Location location) {
         CraftPlayer craftPlayer = (CraftPlayer) player;
-        ((CraftBlock) location.getBlock()).getNMS().use(((CraftWorld) location.getWorld()).getHandle(),
-                craftPlayer != null ? craftPlayer.getHandle() : null, InteractionHand.MAIN_HAND,
+        // TODO: 1.20.6: passing a null player isn't valid (and seemingly never was) - need to require HumanEntity in the mechanism

```

Comment:
Yeah this always required a player. That'd just be a null check on the mech code